tests/ops/api/v1/endpoints/test_privacy_notice_endpoints.py

```diff
@@ -764,6 +765,7 @@ def test_get_privacy_notice_by_data_use_wrong_scope(
                             cookies=[
                                 CookieSchema(name="test_cookie", path="/", domain=None)
                             ],
+                            default_preference=UserConsentPreference.opt_out,
```

Comment:
I have just unified the PrivacyNoticeResponseSchemas, I don't think there are any issues with this default preference being returned here